samples/Arducam/app/application.cpp

```diff
@@ -280,8 +285,8 @@ void init()
 	WifiStation.config(WIFI_SSID, WIFI_PWD);
 	WifiAccessPoint.enable(false);
 
-	// Run our method when station was connected to AP
-	WifiStation.waitConnection(connectOk);
+	WifiEvents.onStationConnect(&connectOk);
```

Comment:
> I am not sure.

Let's say that I would prefer it to be without reference. And thanks for your great help!